src/schema/v2/filterArtworksConnection.ts

```diff
@@ -0,0 +1,134 @@
+import Artwork from "./artwork"
+import {
+  computeTotalPages,
+  createPageCursors,
+  pageToCursor,
+} from "./fields/pagination"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs, removeNulls } from "lib/helpers"
+import { connectionFromArraySlice, connectionDefinitions } from "graphql-relay"
+import { GraphQLFieldConfig } from "graphql"
+
+import { ResolverContext } from "types/graphql"
+import { includesFieldsOtherThanSelectionSet } from "lib/hasFieldSelection"
+import { FilterArtworksFields, filterArtworksArgs } from "./filter_artworks"
+
+// The connection fields here match the original implementation's,
+// except without `artworks_connection`.
+// TODO: Support proper thunk.
```

Comment:
In order to DRY it up, I'm trying to re-use the fields already defined for filter artworks, but removing the `artworks_connection` one. However, that field definition is a thunk (avoiding some circular/diamond dependency stuff), and I can't quite figure out how to share it in this way (since here I'm calling it directly).

Kind of #minor since I can just copy/paste the fields! More just nerd-sniping.

Reply:
Perhaps?

```ts
const thunk = () => {
  const { artworks_connection, ...connectionFields } = FilterArtworksFields()
  return connectionFields
}
```

But copy-pasting might be preferable, or move the definitions here and refer to them from the deprecated version instead? 🤔